deploy-commands.ts

```diff
@@ -0,0 +1,36 @@
+import dotenv from "dotenv";
+dotenv.config();

```

Comment:
you can import dotenv config setup instead of calling a method -> `import "dotnev/config";` best way